articles/quickstart.md

```diff
@@ -162,179 +204,145 @@ A quantum operation transforms the state of a qubit. Sometime people talk about
-                        var (numZeros, numOnes) = res;
-                        System.Console.WriteLine(
-                            $"Init:{initial,-4} 0s={numZeros,-4} 1s={numOnes,-4}");
-                    }
-                }
+```
+namespace Bell {
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Intrinsic;
 
-                System.Console.WriteLine("Press any key to continue...");
-                Console.ReadKey();
-            }
+    operation Set(desired : Result, q1 : Qubit) : Unit {
+        if (desired != M(q1)) {
+            X(q1);
         }
     }
-    ```
-
-#### [](#tab/tabid-vs2019)
-
-* * *
-
-### About the host application code
-
-#### [Python](#tab/tabid-python)
-
-The Python host application has three parts:
-
-* Compute any arguments required for the quantum algorithm. In the example, `count` is fixed at a 1000 and `initial` is the initial value of the qubit.
-* Run the quantum algorithm by calling the `simulate()` method of the imported Q# operation.
-* Process the result of the operation. In the example, `res` receives the result of the operation. Here the result is a tuple of the number of zeros (`num_zeros`) and number of ones (`num_ones`) measured by the simulator. We deconstruct the tuple to get the two fields, and print the results.
-
-#### [C#](#tab/tabid-csharp)
-
-The C# host application has four parts:
-
-* Construct a quantum simulator. In the example, `qsim` is the simulator.
-* Compute any arguments required for the quantum algorithm. In the example, `count` is fixed at a 1000 and `initial` is the initial value of the qubit.
-* Run the quantum algorithm. Each Q# operation generates a C# class with the same name. This class has a `Run` method that **asynchronously** executes the operation. The execution is asynchronous because execution on actual hardware will be asynchronous. Because the `Run` method is asynchronous, we fetch the `Result` property; this blocks execution until the task completes and returns the result synchronously.
-* Process the result of the operation. In the example, `res` receives the result of the operation. Here the result is a tuple of the number of zeros (`numZeros`) and number of ones (`numOnes`) measured by the simulator. This is returned as a ValueTuple in C#. We deconstruct the tuple to get the two fields, print the results, and wait for a keypress.
-
-#### [](#tab/tabid-vs2019)
-
-* * *
 
-## Build and run
-
-#### [Python](#tab/tabid-python)
+    @EntryPoint()
+    operation TestBellState(count : Int, initial : Result) : (Int, Int) {
 
-1. Run the following command at your terminal:
+        mutable numOnes = 0;
+        using (qubit = Qubit()) {
 
-    ```
-    python host.py
-    ```
+            for (test in 1..count) {
+                Set(initial, qubit);
+                let res = M(qubit);
 
-    This command runs the host application, which simulates the Q# operation.
+                // Count the number of ones we saw:
+                if (res == One) {
```

Comment:
This is a comparison of a `Result` value, which may not be supported in some scenarios. May be good to check with @KittyYeungQ on how to resolve that.